test/basic.js

```diff
@@ -33,6 +33,12 @@ assert.equal(string.getAlpha("hwb(244, 100%, 100%, 0.6)"), 0.6);
 assert.equal(string.getAlpha("hwb(244, 100%, 100%)"), 1);
 
 // alpha
+assert.deepEqual(string.getRgba("#feff"), [255, 238, 255, 1]);
```

Comment:
it probably would have been preferable to copy the tests from upstream to keep things in sync as much as possible instead of writing our own